src/workloads/execution/BackgroundTTLDeletions.yml

```diff
@@ -13,20 +13,18 @@ Keywords:
 - indexes
 - insertMany
 - CrudActor
-- tickets
 
 Clients:
   Default:
     QueryOptions:
       maxPoolSize: 256
 
-NumPhases: &NumPhases 20
-CollectionCount: &NumColls 1
+NumPhases: &NumPhases 6
 
 Document: &generatedDoc
-  expireDate: {^Now: {}}  # Instantly expire the document
-  y: &y {^RandomInt: {min: -1000, max: 1000}}
-  b: {^RandomString: {length: 15}}
+  expireDate: {^Date: "1970-01-01"}  # Instantly expire the document
+  y: 999
+  b: "Arbitrary string with 45 different characters"
```

Comment:
Does it matter that this string likely compresses very well since it's the same string over and over again?

Reply:
It's just filler for the document so that there are more bytes to delete. It doesn't really matter that much.